client/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import Button from './Button';
+
+export default {
+  title: 'Button',
+};
+export const Default = () => (
+  <Button variant="transparent"> Button </Button>
+);
+export const Primary = () => (
+  <Button variant="blue">  Button </Button>
+);
+
+export const Light = () => (
+  <Button variant="light"> Button </Button>
+);
+
+export const Success = () => (
+  <Button variant="success">Button</Button>
+);
+
+export const Danger = () => (
+  <Button variant="danger">Button</Button>
+);
+
+export const Yellow = () => (
+  <Button variant="yellow"> Button </Button>
+);
+
+export const Grey = () => (
+  <Button variant="grey"> Button </Button>
+);
+
+export const Bouncy = () => (
+  <Button variant="bouncy"> Button </Button>
+);
+
+export const WithIcon = () => (
+  <Button variant="with-icon"> Button </Button>
+);
+
+export const OutlineDanger = () => (
+  <Button variant="outline btn--danger"> Button </Button>
```

Comment:
Typescript should flag this:

![image](https://user-images.githubusercontent.com/11327761/113253993-b014bf00-92ce-11eb-8a11-45d54ca23038.png)

The variant is wrong. Also, we're gonna need to add `outline` as a prop rather than as a variant since we should be able to have any button variant but with outline.